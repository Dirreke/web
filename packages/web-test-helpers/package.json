{
  "name": "web-test-helpers",
  "private": true,
  "peerDependencies": {
      "@vue/composition-api": "1.7.1",
      "vue-gettext": "^2.1.5",
      "vue-router": "3.6.5",
      "vuex": "^3.6.2"
  }
}
