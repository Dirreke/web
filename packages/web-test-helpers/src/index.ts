export * from './mocks/defaultComponentMocks'
export * from './mocks/defaultStubs'
export * from './mocks/axios'

export * from './defaultPlugins'
export * from './helpers'
