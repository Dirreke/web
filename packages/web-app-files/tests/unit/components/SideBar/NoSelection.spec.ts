import NoSelection from 'web-app-files/src/components/SideBar/NoSelection.vue'
import { defaultPlugins, shallowMount } from 'web-test-helpers'

describe('NoSelection', () => {
  it('should render the component', () => {
    const { wrapper } = getWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})

function getWrapper() {
  return {
    wrapper: shallowMount(NoSelection, {
      global: {
        plugins: [...defaultPlugins()]
      }
    })
  }
}
