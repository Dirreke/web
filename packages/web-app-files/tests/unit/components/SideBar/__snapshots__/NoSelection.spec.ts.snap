// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSelection should render the component 1`] = `
<div class="oc-text-center oc-mt-xl" id="oc-no-selection">
  <oc-icon-stub accessiblelabel="" color="" filltype="line" name="drag-drop" size="xxlarge" type="span" variation="passive"></oc-icon-stub>
  <p>Select a file or folder to view details.</p>
</div>
`;
